tests/lib/rules/require-prop-comment.js

```diff
@@ -0,0 +1,263 @@
+/**
+ * @author *****CZB*****
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/require-prop-comment')
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module'
+  }
+})
+
+tester.run('require-prop-comment', rule, {
+  valid: [
+    {
+      code: `
+      <template>
+        <div>1</div>
+      </template>
```

Comment:
Please remove the template from all tests, it is superfluous in this rule.

Reply:
complete